### ISSUE TITLE
Resource map sometimes fails to render 

### ISSUE THREAD

**Issue by dmullen17** [NCEAS/metacatui#948]
Ran into an error where the resource map will sometimes fail to render data objects on a landing page.  It sends two duplicate 404 error messages: 
```
Failed to load resource: the server responded with a status of 404 (Not Found) https://arcticdata.io/metacat/d1/mn/v2/meta/urn%3Auuid%3Ae8e15e18-8607-45d0-89b1-7c0067b4ffe3
```
And one warning message: 
```
DevTools failed to parse SourceMap: https://arcticdata.io/catalog/components/annotator/v1.2.10/annotator-full.map
```
<img width="1720" alt="screenshot" src="https://user-images.githubusercontent.com/15136438/56915486-7ba58200-6a6b-11e9-8b6f-2070275bf2ec.png">

**Comment by jagoldstein** [comment]
@dmullen17 what are the OS and browser versions?

**Comment by dmullen17** [comment]
Good catch @jagoldstein.  MacOS Sierra 10.12.6 and Chrome 74.0.3729.108

**Comment by laurenwalker** [comment]
When I access the dataset linked in the screenshot above (which is https://arcticdata.io/catalog/view/urn:uuid:462974fb-7f5c-4d64-8012-aca1e38e297c), I get a 404 for this identifier instead:

`https://arcticdata.io/metacat/d1/mn/v2/meta/urn%3Auuid%3A10bd6f78-67af-4a1a-9326-d75725af5287`

It's odd that the same dataset has 404s for different ids, both of which are not ids of objects in the package. This seems to be the source of this intermittent rendering issue.

**Comment by csjx** [comment]
This looks like an issue with caching and rendering.  When I reload the page, the package table items don't render (with caching enabled). When I open the inspector, and disable caching, each time I reload the page, all items render correctly.  Needs a deeper look into what is causing this.

**Comment by robyngit** [comment]
We should see if this is still an issue and work on it if needed after the Folders in Datasets #1450 epic.
